charts/openstack-cloud-controller-manager/templates/daemonset.yaml

```diff
@@ -38,9 +38,9 @@ spec:
             - --cloud-provider=openstack
             - --use-service-account-credentials=true
             {{- if .Values.serviceMonitor.enabled }}
-            - --address=0.0.0.0
+            - --bind-address=0.0.0.0
```

Comment:
Is this needed for the configurable volume mounts?

Reply:
This is not required for this, but the `address` flag has been deprecated. When using `--address`, occm prints the following:
```
Flag --address has been deprecated, see --bind-address instead.
```

Reply:
I leave it to the approvers if they want to take this in on commit.